src/app/components/scrollBar/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { ReactNode } from 'react';
+import styled from 'styled-components';
+
+const ScrollBarContainer = styled.div({
+  overflowY: 'hidden',
+  scrollbarGutter: 'stable',
+  ':hover': {
+    overflowY: 'auto',
+  },
+  '&::-webkit-scrollbar': {
+    width: 8,
+    height: 10,
+    background: 'transparent',
+  },
+  '&::-webkit-scrollbar-thumb': {
+    width: 8,
+    maxHeight: 10,
+    borderRadius: 24,
```

Comment:
is there a way to do this without the scrollbar taking up width? then you wouldn't have to update all the marginRight on each consuming component